### ISSUE TITLE
[Feature] Stable and Receive events checks in fill() method

### ISSUE THREAD

**Issue by Diokuz** [microsoft/playwright#20945]
Hi guys!

We have some problems with test stability in firefox and webkit. Some of them related to timeouts, but some – to scroll position. Sometimes, fill() not happens, and it looks like playwright just doesnt scroll to the field.

To fix that, we call click({ trial: true }) before fill(), and it fixed the test.

According to https://playwright.dev/docs/actionability, click have 2 more checks, particulary [Stable](https://playwright.dev/docs/actionability#stable) and [Receives Events](https://playwright.dev/docs/actionability#receives-events).

What do you think about adding these checks for fill() method?

See also https://github.com/microsoft/playwright/discussions/20486

**Comment by Diokuz** [comment]
No, I can not reproduce it locally. It fails only in CI, from time to time. Probably this happens because CI runners are much slower than my macbook.

When we use fill(), sometimes webkit doesnt fill our input. In trace we see that webkit didnt scrolled input into viewport.

When we add code

```
await page.locator(`xpath=${xpath}`).click({
  trial: true,
})
await page.fill(`xpath=${xpath}`, value)
```

webkit works just fine in 100% test runs.

**Comment by aslushnikov** [comment]
@Diokuz oh actually, this looks very much like the bug that we fixed recently:

https://github.com/microsoft/playwright/issues/20165

So this should be fixed in the upcoming 1.31 or you can try it right now with beta version: `npm i @playwright/test@beta`.

I'll speculatively close as a dupe of https://github.com/microsoft/playwright/issues/20165
